requirements/common.in

```diff
@@ -138,7 +138,7 @@ py3dns
 
 # Install Python Social Auth
 social-auth-app-django
-https://github.com/python-social-auth/social-core/archive/3.4.0.zip/#egg=social-auth-core[azuread,saml]==3.4.0
+https://github.com/python-social-auth/social-core/archive/3.4.0.zip/#egg=social-auth-core[azuread,saml,facebook]==3.4.0
```

Comment:
`facebook` isn’t a [valid extra](https://github.com/python-social-auth/social-core/blob/master/setup.py#L87-L96) for `social-auth-core`.